<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindDialog</class>
 <widget class="QDialog" name="FindDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>114</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find ...</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Find what?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineFind"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QComboBox" name="comboStringType">
       <item>
        <property name="text">
         <string>Contains</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Regular Expression</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Wildcard</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkCaseSensitive">
     <property name="text">
      <string>Case Sensitive</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnFindPrev">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Find &amp;Previous</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnFindNext">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Find &amp;Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>FindDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
